sheet_to_triples/tests/test_rdf.py

```diff
@@ -0,0 +1,242 @@
+# This is free software licensed as GPL-3.0-or-later - see COPYING for terms.
+
+"""Unittests for the rdf.py module of sheet-to-triples."""
+
+import unittest
+
+import rdflib.term
+
+from .. import rdf
+
+
+class TestRDF(unittest.TestCase):
+
+    def test_from_qname(self):
+        term = rdf.from_qname('owl:test')
+        self.assertIsInstance(term, rdflib.term.URIRef)
+        self.assertEqual(str(term), 'http://www.w3.org/2002/07/owl#test')
+
+    def test_from_qname_prefix_vm(self):
+        term = rdf.from_qname('vm:test')
+        self.assertIsInstance(term, rdflib.term.URIRef)
+        self.assertEqual(str(term), 'http://visual-meaning.com/rdf/test')
+
+    def test_from_qname_prefix_http(self):
+        term = rdf.from_qname('http://test.test')
+        self.assertIsInstance(term, rdflib.term.URIRef)
+        self.assertEqual(str(term), 'http://test.test')
+
+    def test_from_identifier_already_ident(self):
+        term = rdflib.term.URIRef('test')
+        self.assertEqual(
+            rdf.from_identifier(term),
+            term
+        )
+
+    def test_from_identifier_sequencepath(self):
+        term = rdf.from_identifier('vm:a / vm:b')
+        expected = [
+            'http://visual-meaning.com/rdf/a',
+            'http://visual-meaning.com/rdf/b',
+        ]
+        self.assertIsInstance(term, rdflib.paths.SequencePath)
+        self.assertEqual(
+            [str(p) for p in term.args],
+            expected
+        )
+
+    def test_from_identifier_prefix_http(self):
+        term = rdf.from_identifier('http://test.test')
+        self.assertIsInstance(term, rdflib.term.URIRef)
+        self.assertEqual(str(term), 'http://test.test')
+
+    def test_from_identifier_literal(self):
+        self.assertEqual(
+            rdf.from_identifier('test'),
+            rdflib.Literal('test')
+        )
+
+    def test_relates_geo_name_true(self):
+        for path in ('atGeoPoint', 'atGeoPoly', 'name'):
+            term = {'pred': 'http://visual-meaning.com/rdf/' + path}
+            with self.subTest(pred=term['pred']):
+                self.assertTrue(rdf.relates_geo_name(term))
+
+    def test_relates_geo_name_false(self):
+        for path in ('http://visual-meaning.com/rdf/test', 'test/atGeoPoint'):
+            term = {'pred': path}
+            with self.subTest(pred=term['pred']):
+                self.assertFalse(rdf.relates_geo_name(term))
+
+    def test_relates_issue_true(self):
+        self.assertTrue(
+            rdf.relates_issue({'subj': 'http://visual-meaning.com/rdf/test'})
+        )
+
+    def test_relates_issue_false(self):
+        self.assertFalse(
+            rdf.relates_issue(
+                {'subj': 'http://visual-meaning.com/rdf/issues/'},
+            )
+        )
+
+    def test_purge_terms(self):
+        model = {
+            'terms': [
+                {'obj': 'http://visual-meaning.com/rdf/test'},
+                {'obj': 'http://visual-meaning.com/rdf/whitespace   '},
```

Comment:
I'd use a non-iri obj value here, as that's what the whitespace normalisation is about.